### PR TITLE
Update about us

### PR DESCRIPTION
**Pull request by Plishh**
fix #14

### CODE REVIEW

File: docs/AboutUs.md [hunk 1]
```diff
@@ -11,51 +11,12 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 
 ## Project team
 
-### John Doe
+### Lincoln Yao
 
-<img src="images/johndoe.png" width="200px">
+<img src="imgaes/plishh.png" width="200px">
```
Comment (Jollybomber) [inline docs/AboutUs.md:16]:
```suggestion
<img src="images/plishh.png" width="200px">
```

**Review by Jollybomber — CHANGES_REQUESTED** [review]

File: docs/AboutUs.md (hunk 1 above)
Comment (Plishh) [reply docs/AboutUs.md]: changed

**Review by Jollybomber — APPROVED** [review]
LGTM

**Comment by mingyang143** [comment]
Resolve merge conflicts thanks.

**Pull request merged**